examples/custom-server-express/pages/posts/[id].js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+function Post({ postId }) {
+  return (
+    <div>
+      <h1>My blog post #{postId}</h1>
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+        tempor incididunt ut labore et dolore magna aliqua.
+      </p>
+    </div>
+  )
+}
+
+export function getStaticProps({ params: { id } }) {
+  return { props: { postId: id } }
+}
+
+export function getStaticPaths() {
+  return { paths: [], fallback: true }
```

Comment:
I wasn't sure if passing an empty array to the `paths` is the way to go but I didn't want to put any hardcoded values here as it wouldn't be clear where they come from. But, of course, I might be mistaken :)

Reply:
In this case I opted for removing the entire page, in the real world `getStaticPaths` will never be used like this.